pants_plugins/sendwave/pants_docker/python_requirement.py

```diff
@@ -93,12 +90,56 @@ async def get_requirements(
     constraint_arg = ""
     if setup.requirement_constraints:
         constraint_arg = f"--constraint {setup.requirement_constraints}"
+    return PipInstallArgs(
+        links_args=links_args,
+        index_args=index_args,
+        constraint_arg=constraint_arg,
+    )
+
+
+@dataclass(frozen=True)
+class PythonRequirementsFS:
```

Comment:
could I suggest renaming this to remove the `FS`, which I think stands for FieldSet, which this is (no longer) a subclass of?

Reply:
Good call, done in 9f7d0fa5e1dc28309e6305be4b68c3d0cc1e529!